### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry/PrimeSpectrum): prime spectrum of jacobson rings

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/PrimeSpectrum/Jacobson.lean [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+open Ideal
+
+variable {R : Type*} [CommRing R] [IsJacobson R]
+
+namespace PrimeSpectrum
+
+/-- The spectrum of a jacobson ring is a jacobson space. -/
+lemma exists_isClosed_singleton_of_isJacobson
+    (s : (Set (PrimeSpectrum R))) (hs : IsOpen s) (hs' : s.Nonempty) :
+    ∃ x ∈ s, IsClosed {x} := by
```
[the remark this answers is not in the document]
Comment (erdOne) [reply Mathlib/AlgebraicGeometry/PrimeSpectrum/Jacobson.lean:32]: Done, I travelled to the past and opened #18243

[the remark this answers is not in the document]
Comment (jcommelin) [reply Mathlib/AlgebraicGeometry/PrimeSpectrum/Jacobson.lean:32]: Awesome. In that case, could you please make this PR depend on that one?

[the remark this answers is not in the document]
Comment (erdOne) [reply Mathlib/AlgebraicGeometry/PrimeSpectrum/Jacobson.lean:32]: I think the two PRs can be reviewed and merged separately though. As this does not immediately imply that `Spec R` is Jacobson (despite the misleading comment, I will remove it) and the equivalence `IsJacobson R <-> JacobsonSpace (Spec R)` will still need to come in a later PR.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**